src/node_task_runner.cc

```diff
@@ -21,6 +17,20 @@ ProcessRunner::ProcessRunner(
     const std::optional<std::string>& positional_args) {
   memset(&options_, 0, sizeof(uv_process_options_t));
 
+  // Get the current working directory.
+  char cwd[PATH_MAX_BYTES];
+  size_t cwd_size = PATH_MAX_BYTES;
+  CHECK_EQ(uv_cwd(cwd, &cwd_size), 0);
+  CHECK_GT(cwd_size, 0);
+
+#ifdef _WIN32
+  options_.file = "cmd.exe";
+  std::string current_bin_path = cwd + std::string(bin_path) + ";";
+#else
+  options_.file = "/bin/sh";
+  std::string current_bin_path = cwd + std::string(bin_path) + ":";
```

Comment:
I don't expect that the call to `std::string` is needed in `std::string(bin_path)`. You should be able to just do `cwd + bin_path + ":"`. I think. Right? Same comment above.

Reply:
This fails because cwd is a fixed size char*?

```
../../src/node_task_runner.cc:31:31: error: invalid operands to binary expression ('char[1024]' and 'const char[19]')
  auto current_bin_path = cwd + bin_path + ":";
```

Reply:
Ah. Yes, but see my other proposals. You can just declare them as C strings and it should all work out.

Reply:
(Note: this does not matter... what you have is likely fine, I am just trying to help simplify it a tiny bit.)

Reply:
Yeah, you're right. Your recommendation is better than the current one. I'll apply them. Thanks!

Reply:
A different problem now:

```
C:\Users\yagiz\Desktop\coding\node\src\node_task_runner.cc(27,38): error C2110: '+': cannot add two pointers [C:\Users\
yagiz\Desktop\coding\node\libnode.vcxproj]
```